app/assets/javascripts/pretty-text/engines/discourse-markdown/emoji.js.es6

```diff
@@ -257,5 +265,10 @@ export function setup(helper) {
     );
   });
 
-  helper.whiteList(["img[class=emoji]", "img[class=emoji emoji-custom]"]);
+  helper.whiteList([
```

Comment:
That feels a bit weird. Isn't there a better way to whitelist 3 classes on the `img` tag?

Reply:
No I don't think there's a better way

Reply:
maybe @SamSaffron  can advise here, but AFAIK you have to define each combination